backend/src/ml_space_lambda/group/lambda_functions.py

```diff
@@ -191,11 +228,24 @@ def delete(event, context):
     # Delete users associations and group itself
     to_delete_group_users = group_user_dao.get_users_for_group(group_name)
 
+    # Remove group from all projects
+    for project_group in project_group_dao.get_projects_for_group(group_name):
+        project_group_dao.delete(project_group.project, group_name)
+
+        if env_variables[EnvVariable.MANAGE_IAM_ROLES]:
+            for group_user in to_delete_group_users:
+                if not is_member_of_project(group_user.user, project_group.project):
```

Comment:
I think we want to remove the `not` here so that we are updating the policies and removing the project role from users that are in the project

Reply:
I think it is right. After deleting the `project_user` association on line 233 it only deletes their dynamic role (user/project combination) if they're not a member of the project through any other means (another group or directly added)

Reply:
Ah, got ya.